### ISSUE TITLE
Audio doesent work anymore. No speaker in desktop mode 

### ISSUE THREAD

**Issue by Quecks87** [ValveSoftware/SteamOS#1360]
#### Your system information

* Steam client version: up 2 date - watch later 
* SteamOS version: up 2 date - watch later 
* Opted into Steam client beta?: [No] 
* Opted into SteamOS beta?: [No] 
* Have you checked for updates in Settings > System?: [Yes]

#### Please describe your issue in as much detail as possible:

Hi @ all first! 🙃
I have from one to the other day no sound  over the speaker or headphones anymore. Bluetooth too not connect.  By the first time right volume doesent work.. After some reset it work 

In desktop mode I see I have no speaker to select. I try many thinks. Reset. Old or new bios 116 or current 119. 

It is not the little mute 🔕 symbol in the edge of the game icon by the Taskbar. 

I think maybe the board it's broken or software problem? I change the backplate sometime with the jsaux.. But I think it's not the problem..I don't hit anything inside. 

#### Steps for reproducing this issue:

1. Just no sound 🔊 
![Screenshot_2024-01-19-06-49-41-56_a23b203fd3aafc6dcb84e438dda678b6](https://github.com/ValveSoftware/SteamOS/assets/157163009/1b1b3eb6-b89f-41a5-8ec4-6836e2bd5450)

Some consol info

deck@steamdeck ~)$ aplay -L
null
Discard all samples (playback) or generate zero samples (capture)
lavrate
Rate Converter Plugin Using Libav/FFmpeg Library
samplerate
Rate Converter Plugin Using Samplerate Library
speexrate
Rate Converter Plugin Using Speex Resampler
jack
JACK Audio Connection Kit
oss
Open Sound System
pipewire
PipeWire Sound Server
pulse
PulseAudio Sound Server
speex
Plugin using Speex DSP (resample, agc, denoise, echo, dereverb)
upmix
Plugin for channel upmix (4,6,8)
vdownmix
Plugin for channel downmix (stereo) with a simple spacialization
default
Default ALSA Output (currently PipeWire Media Server)
hdmi:CARD=Generic,DEV=0
HD-Audio Generic, HDMI 0(deck@steamdeck ~)$ aplay -L
null
Discard all samples (playback) or generate zero samples (capture)
lavrate
Rate Converter Plugin Using Libav/FFmpeg Library
samplerate
Rate Converter Plugin Using Samplerate Library
speexrate
Rate Converter Plugin Using Speex Resampler
jack
JACK Audio Connection Kit
oss
Open Sound System
pipewire
PipeWire Sound Server
pulse
PulseAudio Sound Server
speex
Plugin using Speex DSP (resample, agc, denoise, echo, dereverb)
upmix
Open Sound System
Plugin for channel upmix (4,6,8)
vdownmix
Plugin for channel downmix (stereo) with a simple spacialization
default
Default ALSA Output (currently PipeWire Media Server)
Open Sound System
Open Sound System
hdmi:CARD=Generic,DEV=0
HD-Audio Generic, HDMI 0
HDMI Audio Output
hdmi:CARD=Generic,DEV=1
HD-Audio Generic, HDMI 1
HDMI Audio Output
hdmi:CARD=Generic,DEV=2
HD-Audio Generic, HDMI 2
HDMI Audio Output
Open Sound System
hdmi:CARD=Generic,DEV=3
HD-Audio Generic, HDMI 3
HDMI Audio Output
usbstream:CARD=Generic
HD-Audio Generic
USB Stream Output
(deck@stea
HDMI Audio Output
hdmi:CARD=Generic,DEV=1
HD-Audio Generic, HDMI 1
HDMI Audio Output
hdmi:CARD=Generic,DEV=2
HD-Audio Generic, HDMI 2
HDMI Audio Output
hdmi:CARD=Generic,DEV=3
HD-Audio Generic, HDMI 3
HDMI Audio Output
usbstream:CARD=Generic
HD-Audio Generic
USB Stream Output
(deck@stea

**Comment by GreatNateDev** [comment]
Well is there a fix?

**Comment by Quecks87** [comment]
> Well is there a fix?

I don't find something.. Valve support don't answer anymore 😭.. The first answer was only simpel questions

**Comment by Quecks87** [comment]
No answer.. No ideas.. I try many workaroundas..the last change pipewire parts over the consol.. I become back the show inactive devices but it's a empty option dummy I can't select..

**Comment by GreatNateDev** [comment]
Fixed in newest steam preview:)

**Comment by Quecks87** [comment]
> Fixed in newest steam preview:)

What do u mean? How fix?

**Comment by GreatNateDev** [comment]
In the system settings in gamemode change to preview instead of stable and once reboot check for updates and update and that worked for me

**Comment by GreatNateDev** [comment]
Seemed to be fixed since Jan 19 💀

**Comment by GreatNateDev** [comment]
Did you ever test the preview branch? And does it work for you now?

**Comment by GreatNateDev** [comment]
> Did you ever test the preview branch? And does it work for you now?

Maybe we have diff but similar audio issues?

**Comment by Quecks87** [comment]
> > Did you ever test the preview branch? And does it work for you now?
> 
> Maybe we have diff but similar audio issues?

Maybe.. I have no sound with this preview 😭

**Comment by Quecks87** [comment]
I have no speaker in my options. So sad.. 
![Uploading 17059462452425131630504324640482.jpg…]()

**Comment by GreatNateDev** [comment]
Dang man I'm sorry to hear that you can send the deck in to valve I think or maybe try a full reimage

**Comment by GreatNateDev** [comment]
I had the no speaker thing to so weird that mine got fixed I gtg for now though

**Comment by Quecks87** [comment]
> Dang man I'm sorry to hear that you can send the deck in to valve I think or maybe try a full reimage

I have try complete full image over the USBc stick.. I test every bios version.. I try consolcomands... I write with valve.. This device go back..😭

**Comment by GreatNateDev** [comment]
That sucks bro maybe it is a hardware issue

**Comment by Quecks87** [comment]
> Did you ever do hardware mods eg:replace ssd

Nope...

**Comment by GreatNateDev** [comment]
Idk that sounds like a hardware issue maybe something broke send to valve

**Comment by Quecks87** [comment]
> Idk that sounds like a hardware issue maybe something broke send to valve

Yeah... But I don't know.. This device run only 10 hour.. And have this problems.. Maybe I get the refund and buy another... So sad

**Comment by Quecks87** [comment]
> Idk that sounds like a hardware issue maybe something broke send to valve

With pipewire commend I can get the auto null bug and the empty dummy sound button.. 🤔😕 Co u know maybe fix?

**Comment by GreatNateDev** [comment]
I got that too when I had bug couldn't get passed that stage tho

**Comment by hungrymonkey** [comment]
Post the output of `dmesg`
??

**Comment by GreatNateDev** [comment]
You want me to do that @Quecks87 ?
